grpc/oob/machine/machine.go

```diff
@@ -251,14 +295,17 @@ func (m Action) PowerSet(ctx context.Context, action string) (result string, err
 	}
 	log = m.Log.WithValues(logMetadata(client.GetMetadata())...)
```

Comment:
The data from client.GetMetadata() would be very valuable for annotating the trace.